### PR TITLE
update utils subworkflows

### PR DESCRIPTION
**Pull request by maxulysse**
replace #4766
copy files from fetchngs:dev with minor code polishing

### CODE REVIEW

**Review by drpatelh — APPROVED** [review]
LGTM!

File: subworkflows/nf-core/utils_nextflow_pipeline/main.nf [hunk 1]
```diff
@@ -15,10 +15,10 @@ import nextflow.extension.FilesEx
 workflow UTILS_NEXTFLOW_PIPELINE {
 
     take:
-    print_version           // boolean: print version
-    dump_parameters         // boolean: dump parameters
-    outdir                  // path: base directory used to publish pipeline results
-    check_conda_channels    // boolean: check conda channels
+    print_version        // boolean: print version
+    dump_parameters      // boolean: dump parameters
+    outdir               //    path: base directory used to publish pipeline results
```
Comment (mashehu) [inline subworkflows/nf-core/utils_nextflow_pipeline/main.nf:20]: that's a new rule of the [Harshil Alignment™️](https://nf-co.re/docs/contributing/modules#what-is-the-harshil-alignment), no?

**Review by mashehu — APPROVED** [review]

File: subworkflows/nf-core/utils_nextflow_pipeline/main.nf (hunk 1 above)
Comment (mashehu) [reply subworkflows/nf-core/utils_nextflow_pipeline/main.nf:20]: cc @jfy133

Comment (jfy133) [reply subworkflows/nf-core/utils_nextflow_pipeline/main.nf:20]: *sigh* you made me do it... https://github.com/nf-core/website/pull/2205

**Pull request merged**